### PR TITLE
feat(issue-203): create accountant and sales person middleware and seeders

### PR DESCRIPTION
**Pull request by mandocino**
- Create accountant specific sections within the navbar and prevents other users to view accountant page but the accountants.
- Create sale specific sections within the navbar and prevents other users to view sales page but the sales worker.

### CODE REVIEW

File: ERP/resources/views/components/navbar.blade.php [hunk 1]
```diff
@@ -66,6 +57,17 @@
...
+                
+                @if(Auth::user()->user_type==6)
+                <!--Redirects to Accountant page if the user has permission-->
+                <li class="nav-item">
+                    <a class="nav-link active" href="{{route('accountant')}}">Accountant</a>
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link active" href="{{route('sales')}}">Sales</a>
+                </li>
```
Comment (CeliaCaii) [inline ERP/resources/views/components/navbar.blade.php:69]: Accountant does not have access to Sales, it is the sales person that has access to Sales page.
The endif should end at line 66

File: ERP/routes/web.php [hunk 2]
```diff
@@ -137,20 +137,26 @@
         ->name('toggle.order.status');
 });
 
-// Executes "goToAccoutantView" method in the AccountantController when the route is "/accountant".
-Route::get('/accountant', [AccountantController::class, 'goToAccoutantView'])
-    ->name('accountant');
+//Sales Routes given `Accountant.access.only` middleware (prevents non-accountant or non-IT personal from accessing this route)
```
Comment (CeliaCaii) [inline ERP/routes/web.php:140]: I don't think IT personal have access to the accountant page

File: ERP/routes/web.php [hunk 3]
```diff
@@ -137,20 +137,26 @@
...
+
+    Route::post('/sales', [SaleController::class, 'saveSaleOrder'])
+        ->name('save.sale.order');
+
+    // Executes Accountant export functions (csv and PDF)
+    Route::get('/sales-CSV-export', [AccountantController::class, 'exportSalesCSV'])
+        ->name('saleCSV.export');
+
+    Route::get('/sales-PDF-export', [AccountantController::class, 'exportSalesPDF'])
+        ->name('salePDF.export');
```
Comment (CeliaCaii) [inline ERP/routes/web.php:159]: Remove Sales related pages from the accountant.access.only middleware

**Review by CeliaCaii — CHANGES_REQUESTED** [review]
I left some comments regarding Sales. Sales should be accessible by sales person, but NOT accountant.
Also, I think IT personal cannot access Accountant page either

**Review by CeliaCaii — APPROVED** [review]
Looks good

**Review by mlee97 — APPROVED** [review]

**Pull request merged**